doc/source/examples/rpt/tuning-parameters-with-nomad/tuning-parameters-with-nomad.rst

```diff
@@ -0,0 +1,320 @@
+.. role:: raw-html(raw)
+    :format: html
+
+=======================================================
+Tuning Count Calculation Model Parameters with NOMAD
+=======================================================
+
+
+In this example, `NOMAD <https://www.gerad.ca/en/software/nomad/>`_, the blackbox optimization software is used to estimate the unknown variables of the `Beam et al. (1978) <https://www.sciencedirect.com/science/article/abs/pii/0029554X78900812?via%3Dihub>`_ model. The three unknowns of our studied system are:
+
+- the detector's ``dead time`` (:math:`\tau`)
+- the source's ``activity`` (:math:`R`), and
+- the reactor's attenuation coefficient (``attenuation coefficient reactor`` (:math:`\mu_r`)).
+
+
+:raw-html:`<br />`
+
+Features
+----------------------------------
+- Solver: ``rpt_3d``
+- Displays the use of NOMAD in parameter tuning
+
+
+:raw-html:`<br />`
+
+Locations of files used in the example
+---------------------------------------
+- Parameter file: ``examples/rpt/parameters_tuning/rpt_parameters.prm``
+- Python script for NOMAD: ``examples/rpt/parameters_tuning/rpt_lethe_nomad.py``
+- Text file used when running NOMAD: ``examples/rpt/parameters_tuning/param_nomad.txt``
+- File containing particle positions: ``examples/rpt/parameters_tuning/real_positions.particle``
```

Comment:
```suggestion
- File containing particle positions: ``examples/rpt/parameters_tuning/positions.particle``
```

Reply:
I think you can change all the "real-position" to "position". We changed it to "real-..." before since we had to study the noisy cases.